cetloxblock/cetloxblock.sln

```diff
@@ -0,0 +1,28 @@
+﻿

```

Comment:
When you make a `.gitignore` file, this should also be ignored if it isn't already, i.e. `*.sln` files.